orbitize/nbody.py

```diff
@@ -16,104 +28,115 @@ def calc_orbit(
         inc (np.array): inclination [radians]
         aop (np.array): argument of periastron [radians]
         pan (np.array): longitude of the ascending node [radians]
-        tau (np.array): epoch of periastron passage in fraction of orbital period 
+        tau (np.array): epoch of periastron passage in fraction of orbital period
             past MJD=0 [0,1]
         plx (np.array): parallax [mas]
-        mtot (np.array): total mass of the two-body orbit (M_* + M_planet) 
+        mtot (np.array): total mass of the two-body orbit (M_* + M_planet)
             [Solar masses]
         tau_ref_epoch (float, optional): reference date that tau is defined with
             respect to
         m_pl (np.array, optional): mass of the planets[units]
         output_star (bool): if True, also return the position of the star
             relative to the barycenter.
+        integrator (str): value to set for rebound.sim.integrator. Default "ias15"
 
     Returns:
         3-tuple:
 
-            raoff (np.array): array-like (n_dates x n_orbs) of RA offsets between 
+            raoff (np.array): array-like (n_dates x n_bodies x n_orbs) of RA offsets between
```

Comment:
is n_orbs always 1? if so, we should specific that.

Reply:
yes, good catch